lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -53,15 +53,18 @@
 
 
 @retry(tries=10, jitter=(1, 2))
-def _generate_vhd_path(container_client: Any, file_name_part: str = "") -> str:
-    path = PurePosixPath(
-        f"{get_date_str()}/{get_datetime_path()}_"
-        f"{DEFAULT_VHD_SUFFIX}_{file_name_part}.vhd"
+def _generate_vhd_path(
```

Comment:
Let it generate one path like before, and use replace to generate vmgs path like when it's used.